frontend/components/dm-list/dm_list_item.jsx

```diff
@@ -14,24 +14,23 @@ class DmListItem extends React.Component{
   }
 
   handleSelection(){
-    const dm = this.props.dm;
-    const permissions = this.props.permissions;
-    if (!this.props.dm.private || permissions[dm.id])
-    { this.props.selectChannel(dm.id); }
+    const { dm, selectChannel } = this.props;
```

Comment:
same thing as above. you have consistency now, so I imagine you're choosing to always destructure, and never reach into `props` directly. like i said, preference thing. up to you.